src/java.xml/share/classes/com/sun/org/apache/xalan/internal/xsltc/compiler/sym.java

```diff
@@ -92,9 +90,11 @@ public class sym {
   /*
     AXES: count once at DCOLON,
           these axes names are therefore not counted:
-            NAMESPACE, FOLLOWINGSIBLING, CHILD, DESCENDANTORSELF, DESCENDANT
-          , PRECEDINGSIBLING, SELF, ANCESTORORSELF, PRECEDING, ANCESTOROR, PARENT, FOLLOWING, ATTRIBUTE
+          NAMESPACE, FOLLOWINGSIBLING, CHILD, DESCENDANTORSELF, DESCENDANT,
+          PRECEDINGSIBLING, SELF, ANCESTORORSELF, PRECEDING, ANCESTOROR, PARENT,
+          FOLLOWING, ATTRIBUTE
   */
-  public static final int[] OPERATORS = {GE, SLASH, ATSIGN, LPAREN, DCOLON,
-      MINUS, STAR, LT, OR, DIV, PLUS, LE, VBAR, MOD, EQ, LBRACK, DOLLAR, NE, GT};
+  public static final int[] OPERATORS = {GT, GE, EQ, NE, LT, LE, SLASH, DSLASH,
+      DOT, DDOT, ATSIGN, DCOLON, PLUS, MINUS, STAR, DIV, MOD, AND, OR, LPAREN,
+      LBRACK, VBAR, DOLLAR, NODE, TEXT, PI, PIPARAM};
```

Comment:
Any reason for re-shuffling the order of operators? I'd expect new ones are appended to the existing ones, or appear in the order of their declarations above?
(or is this automatically generated, as described in the comment?)

Reply:
The order is not significant for this process as the lexer takes care of creating the right symbol. I re-grouped them to put operators of the same category together so that it's easier to see what might be missing.